src/components/content-tab-snapshots.tsx

```diff
@@ -150,6 +151,8 @@ function SnapshotRow( {
 						selectedSite={ selectedSite }
 						isSnapshotLoading={ snapshot.isLoading }
 						tagline={ __( "We're creating your new demo site." ) }
+						isExpired
```

Comment:
I wonder if the Clear expired site button should be rendered at the same level as `AddDemoSiteWithProgress` as both are buttons. WDYT?

Reply:
I think it makes sense - I made the adjustment here: https://github.com/Automattic/studio/pull/218/commits/7235e394104ae6aadd18cca95d3384d5ae0b9d14